src/lib/packages.ts

```diff
@@ -136,6 +168,14 @@ export async function hoist(packageName: string): Promise<void> {
   await writeRootPackage(rootPkg);
 }
 
+export namespace hoist {
+  /**
+   * Options for the hoist function
+   */
+  export interface Options {
+    risky?: boolean;
```

Comment:
This is similar to what semver calls "loose", but I like this name way better because it connotes a certain price to pay for being permissive.

Reply:
Yea. I wanted it to be clear that we’ll try to do the right thing for *our* definition of “the right thing”.